test_data/makedata_suites/060_computed_values.js

```diff
@@ -0,0 +1,776 @@
+    `for doc in ${view[0]} search doc.cv_field_update == SOUNDEX('beer') collect with count into c return c`,
+    `for doc in ${view[0]} search doc.cv_field_replace == SOUNDEX('water') collect with count into c return c`,
+    `for doc in ${view[0]} filter doc.cv_field == to_hex(doc.name) collect with count into c return c`,
+    `for doc in ${view[0]} filter doc.cv_field == CONCAT('42_', TO_STRING(doc.field)) collect with count into c return c`,
+    `for doc in ${view[0]} search doc.cv_field1=='foo' and doc.cv_field2=='bar' and doc.cv_field3=='baz' collect with count into c return c`,
+    `for doc in ${view[0]} filter doc.cv_field == CONCAT(doc._key, ' ', doc._id, ' ', doc._rev) collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field == SOUNDEX('sky') collect with count into c return c`, 
+    `for doc in ${view[1]} search doc.cv_field == SOUNDEX('dog') collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field_insert == SOUNDEX('frog') collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field_update == SOUNDEX('beer') collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field_replace == SOUNDEX('water') collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field == null collect with count into c return c`,
+    `for doc in ${view[1]} filter doc.cv_field == to_hex(doc.name) collect with count into c return c`,
+    `for doc in ${view[1]} filter doc.cv_field == CONCAT('42_', TO_STRING(doc.field)) collect with count into c return c`,
+    `for doc in ${view[1]} search doc.cv_field1=='foo' and doc.cv_field2=='bar' and doc.cv_field3=='baz' collect with count into c return c`,
+    `for doc in ${view[1]} filter doc.cv_field == CONCAT(doc._key, ' ', doc._id, ' ', doc._rev) collect with count into c return c`
+  ];
+
+  // Expected output array for the TestView views query output
+  let views_exp_output = [64000, 64000, 64000, 0, 0, 64000, 96000, 64000, 64000, 64000, 64000, 64000, 0, 0, 64000, 64000, 96000, 64000, 64000];
+
+  return [views_array, views_exp_output];
+}
+
+
+(function () {
+  const a = require("@arangodb/analyzers");
+  return {
+    isSupported: function (currentVersion, oldVersion, options, enterprise, cluster) {
+      let oldVersionSemver = semver.parse(semver.coerce(oldVersion));
+      return semver.gte(oldVersionSemver, "3.10.0");
+    },
+
+    makeDataDB: function (options, isCluster, isEnterprise, database, dbCount) {
+      // All items created must contain dbCount
+      print(`060: making per database data ${dbCount}`);
+      print("060: Creating computed values with sample collections");
+
+      // getting all the collection names
+      let c = collection_declaration(dbCount);
+
+      let a1 = createCollectionSafe(c[0], 3, 3, { computedValues: [{ "name": "default", "expression": "RETURN SOUNDEX('sky')", overwrite: true }] });
```

Comment:
Please rename `a` to `c`. And use same number as you use for array `c`.
Should be:

```
let c0 = createCollectionSafe(c[0], 3, 3, { computedValues: [{ "name": "default", "expression": "RETURN SOUNDEX('sky')", overwrite: true }] });
```

And so on